src/wallet/rpcwallet.cpp

```diff
@@ -3361,7 +3361,7 @@ static UniValue bumpfee(const JSONRPCRequest& request)
             if (fee_rate <= 0) {
                 throw JSONRPCError(RPC_INVALID_PARAMETER, strprintf("Invalid feeRate %s (must be greater than 0)", fee_rate.ToString()));
             }
-            coin_control.m_feerate = fee_rate; 
+            coin_control.m_feerate = fee_rate;
```

Comment:
Nit: move whitespace fix to commit where the whitespace mistake was made

Reply:
fixed

Reply:
fixed